query-engine/docker-compose.yaml

```
@@ -0,0 +1,77 @@
+# docker-compose.yml  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)